crates/core/src/chunker.rs

```diff
@@ -16,8 +16,8 @@ pub(super) mod constants {
     pub(super) const MIN_SIZE: usize = 512 * KB;
     /// The maximum size of a chunk.
     pub(super) const MAX_SIZE: usize = 8 * MB;
-    /// Buffer size used for reading.
-    pub(super) const BUF_SIZE: usize = 64 * KB;
+    /// Buffer size used for reading - TODO: Find out optimal size for best performance!
+    pub(super) const BUF_SIZE: usize = 4 * KB;
```

Comment:
common kernel PAGE_SIZE would also be my **guess** ^^

Reply:
Yes, but we don't have performance tests yet. Maybe there is a better one.
Also, we could find out the performance results differ on different systems/plattforms. Then we might want to expose this as a parameter to users..